### ISSUE TITLE
v0.3.16 error win32 x64

### ISSUE THREAD

**Issue by alpha1134** [microsoft/vscode-java-installer#267]
:warning:Please attach **%TEMP%\vscode-java-installer\log.txt** and **%TEMP%\vscode-java-installer\AdoptOpenJDK-MSI.log** for more details.

version: 0.3.16
env:
```
JAVA_HOME: undefined
PATH:
C:\Program Files (x86)\Intel\iCLS Client\
C:\Program Files\Intel\iCLS Client\
C:\WINDOWS\system32
C:\WINDOWS
C:\WINDOWS\System32\Wbem
C:\WINDOWS\System32\WindowsPowerShell\v1.0\
C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL
C:\Program Files\Intel\Intel(R) Management Engine Components\DAL
C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT
C:\Program Files\Intel\Intel(R) Management Engine Components\IPT
C:\WINDOWS\System32\OpenSSH\
C:\MinGW\bin
C:\Program Files\dotnet\
C:\Users\ombha\AppData\Local\Microsoft\WindowsApps

C:\Users\ombha\.dotnet\tools
C:\Users\ombha\AppData\Local\Programs\Microsoft VS Code\bin
JDKs:

```
log:
```
[10:08:48 PM] AdoptOpenJDK-11-x64
[10:09:54 PM] Download JDK ... [Completed]
[10:09:54 PM] Install JDK
[10:09:54 PM] Installing AdoptOpenJDK-11-x64...
[10:10:21 PM] Install JDK ... [Completed]
[10:10:21 PM] Configure Java Settings
[10:10:22 PM] Error occurred
[10:10:22 PM] Error: Java Home not found.

```

**Comment by alpha1134** [comment]
How to solve this??

**Comment by Eskibear** [comment]
it's fixed in v0.3.17, please download from https://aka.ms/vscode-java-installer-win
